test/libsolidity/ASTJSON/used_errors.json

```diff
@@ -1,231 +0,0 @@
-{
```

Comment:
This file being removed happens due to the test file containing analysis errors - I recently reported this as https://github.com/ethereum/solidity/issues/12621
But in this case here: is it intentional that there are analysis errors in this test? Probably not (I haven't looked at the input myself)? Then we should fix them.

Reply:
I restored the contents of the file after recent code changes ( as an experiment ) and the tests  ran perfectly. Have added in the latest push to the PR